nncf/torch/graph/operator_metatypes.py

```diff
@@ -55,6 +55,7 @@ class PTOperatorMetatype(OperatorMetatype):
         NamespaceTarget.TORCH_NN_FUNCTIONAL: [],
```

Comment:
I would suggest to validate this PR using PTQ test scope.

Reply:
Conformance test 419 have finished successfully